P0009/P0009.bs

```diff
@@ -1744,50 +1744,76 @@ template<class OtherExtents>
 <br/>
 <br/>
 
-<b>26.7.� Accessor [mdspan.accessor]</b>
+<b>26.7.� Accessor Policy [mdspan.accessor]</b>
 
-<br/>
-<b>26.7.�.1 Accessor requirements [mdspan.accessor.reqs]</b>
+An *accessor policy* defines types and operations by which
```

Comment:
An accessor is a thing for which `AccessorPolicy::pointer` (current name) is the type.
@crtrott and I discussed several names.
- `pointer` because a pointer can fulfil the operational semantics of an accessor, but does more and thus leads to confusion such as a all semantics of a random access iterator are required of an accessor.
- `accessor` as the more general name, but confusion with the `AccessorPolicy` label, sort of like confusing `iterator` with `iterator_traits`.
- `smart_ptr` because in the IS working draft smart pointers somewhat fulfil accessor semantics; e.g., have `operator[]` and can give access to a contiguous set of elements.